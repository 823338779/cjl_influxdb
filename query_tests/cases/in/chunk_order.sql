-- Test that deduplication respects chunk ordering
-- IOX_SETUP: ChunkOrder

-- query data
SELECT * from cpu order by time;
