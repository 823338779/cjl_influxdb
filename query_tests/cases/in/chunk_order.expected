-- Test Setup: ChunkOrder
-- SQL: SELECT * from cpu order by time;
+--------+--------------------------------+------+
| region | time                           | user |
+--------+--------------------------------+------+
| west   | 1970-01-01T00:00:00.000000100Z | 2    |
+--------+--------------------------------+------+
